src/assets/src/components/edit.tsx

```diff
@@ -4,16 +4,15 @@ import { Link } from "react-router-dom";
 import * as ReactGA from "react-ga";
 import { Modal, Button, Table, Alert, Form } from "react-bootstrap";
 import Dialog from "react-bootstrap-dialog";
-import PhoneInput from "react-phone-input-2";
```

Comment:
If it's not doing anything, I'd rather not see it. It will be easy to remember to re-add this when the phone input is re-included, because the code won't compile without it.